@ekrathSNL the CI is failing, can you please fix it? 
https://github.com/Pressio/rom-tools-and-workflows/actions/runs/7993478483/job/21829392031?pr=155

this is the error we get:
```
************* Module romtools.workflows
romtools/workflows/__init__.py:54:0: E0401: Unable to import 'workflows.dakota' (import-error)
romtools/workflows/__init__.py:55:0: E0401: Unable to import 'workflows.greedy' (import-error)
romtools/workflows/__init__.py:56:0: E0401: Unable to import 'workflows.sampling' (import-error)
************* Module romtools.workflows.greedy.run_greedy
```

--- a/romtools/workflows/__init__.py
+++ b/romtools/workflows/__init__.py
@@ -50,7 +50,6 @@ The workflows module contains all of our "outerloop" workflows used for ROM cons
 - Coupling classes to Dakota for 
   - Random sampling
 '''
-__all__ = ['dakota', 'greedy', 'sampling']
-from workflows.dakota import *
-from workflows.greedy import *
-from workflows.sampling import *
+from romtools.workflows.dakota import *
+from romtools.workflows.greedy import *
+from romtools.workflows.sampling import *

